modules/phase_field/include/ics/MultiSmoothSuperellipsoidIC.h

```diff
@@ -31,29 +31,50 @@ class MultiSmoothSuperellipsoidIC : public SmoothSuperellipsoidBaseIC
 
   virtual void initialSetup();
 
-
 protected:
   virtual void computeSuperellipsoidSemiaxes();
   virtual void computeSuperellipsoidCenters();
   virtual void computeSuperellipsoidExponents();
+  virtual Real overlapCheck(const Point & newcenter, const Real & _nc_as, const Real & _nc_bs, const Real & _nc_cs, const Real & _nc_ns);
 
+  unsigned int _numtries;
   unsigned int _numbub;
   Real _bubspac;
-
-  unsigned int _numtries;
   Real _exponent;
   Real _semiaxis_a;
   Real _semiaxis_b;
   Real _semiaxis_c;
   Real _semiaxis_a_variation;
   Real _semiaxis_b_variation;
   Real _semiaxis_c_variation;
+  Real _nv;
```

Comment:
There's a bunch of new parameters added here (`_bsv`, `_ev`, etc.) with no documentation that also do not seem to be initialized in the constructor.